baremaps-geoparquet/src/main/java/org/apache/baremaps/geoparquet/hadoop/GeoParquetGroupWriter.java

```diff
@@ -17,46 +17,13 @@
 
 package org.apache.baremaps.geoparquet.hadoop;
 
-import org.apache.baremaps.geoparquet.data.GeoParquetGroupImpl;
+import org.apache.baremaps.geoparquet.common.GroupWriter;
 import org.apache.parquet.io.api.RecordConsumer;
 import org.apache.parquet.schema.GroupType;
-import org.apache.parquet.schema.Type;
 
-public class GeoParquetGroupWriter {
-
-  private final RecordConsumer recordConsumer;
-  private final GroupType schema;
+public class GeoParquetGroupWriter extends GroupWriter {
```

Comment:
Can we remove one of the two GeoParquetGroupWriter instead of introducing a parent class? I probably made an error while copying the original example.

Reply:
I was sure if we were not going to need them with new Group support. And I will do another PR for the refactoring. I was thinking we should remove all the dead code at the same time once we have all the support we need for GeoParquet.

Reply:
Yes, we can do this later in another PR.